### ISSUE TITLE
Directly install pytorch (without cpu version) could introduce cuda warnings. 

### ISSUE THREAD

**Issue by yushan111** [intel-analytics/analytics-zoo#3540]
Our document in [orca-pytorch-quickstart](https://analytics-zoo.readthedocs.io/en/latest/doc/Orca/QuickStart/orca-pytorch-quickstart.html) suggests user to install torch with
```bash
pip install torch==1.7.1 torchvision==0.8.2
```
However it could introduce warnings below during `loss.backward()` since the default torch is with cuda.
```
/home/arda/anaconda3/envs/zoo-shan/lib/python3.6/site-packages/torch/autograd/__init__.py:132: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /pytorch/c10/cuda/CUDAFunctions.cpp:100.)
  allow_unreachable=True)  # allow_unreachable flag
```

**Comment by liangs6212** [comment]
I have installed these two packages, but there will still be an error in the first run, but the second run will not prompt an error. 
```python
import warnings
warnings.filterwarnings('ignore')
```
I used this method to correct the error, but the disadvantage is that all warnings are ignored

**Comment by yushan111** [comment]
You can install torch and torchvision with
```bash
conda install pytorch torchvision cpuonly -c pytorch
```
to see whether the warning is gone.

**Comment by liangs6212** [comment]
thanks for your help
